sqldelight-gradle-plugin/build.gradle

```diff
@@ -76,9 +76,8 @@ dependencies {
   implementation libs.kotlin.plugin
   compileOnly libs.android.plugin
 
-  testImplementation libs.sqlPsi
-  testImplementation project(':sqlite-migrations')
-  testImplementation project(':sqldelight-compiler')
+  implementation "net.java.dev.jna:jna:5.9.0"
```

Comment:
could this be moved to libs.versions - is it needed at all?

Reply:
Sure, will do it later. Needed when running the Android connected test because the jna dependency comes from IntelliJ but this version (5.6) does not work with android (in this setup).

Reply:
could it be `testImplementation` then?

Reply:
Nope. Intellij depends on jna 5.6, so the sqldelight plugin uses jna too. But there was a very strange problem when running the android connected device tests: The integration tests failed during executing the sqldelight gradle plugin, when starting the intellij environment, only with this setup: 
`Native library (com/sun/jna/darwin-x86-64/libjnidispatch.jnilib) not found in resource path` (https://github.com/cashapp/sqldelight/pull/3621#issuecomment-1293604322)
There was a change in jna 5.9 resolving the platform names so I updated jna to this version and now the tests pass.

Reply:
Do you want a comment too?